src/platform/packages/shared/kbn-scout/src/servers/flags.ts

```diff
@@ -21,15 +21,20 @@ export const SERVER_FLAG_OPTIONS: FlagOptions = {
   boolean: ['stateful', 'logToFile'],
   help: `
     --stateful           Start Elasticsearch and Kibana with default ESS configuration
-    --serverless         Start Elasticsearch and Kibana with serverless project configuration: es | oblt | security
+    --serverless         Start Elasticsearch and Kibana with serverless project configuration: es | oblt | oblt-logs-essentials | security
```

Comment:
You need to update the CI script in order to run Logs Essentials tests:
https://github.com/elastic/kibana/blob/e2fb58bd00f2d3a8b6aaa6b81c23e1dea4b701ee/.buildkite/scripts/steps/test/scout_configs.sh#L52-L57

Reply:
Done